tests/robot-tests/tests/libs/admin_api.py

```diff
@@ -161,6 +162,40 @@ def user_adds_publication_role_to_user_via_api(
         f'Adding publication role to user API request failed with {response.status_code} and {response.text}'
 
 
+def user_removes_roles_from_prerelease_user_via_api(user_id: str) -> None:
+    response = admin_client.delete(
+        f'/api/user-management/user/{user_id}/release-role/pre-release'
+    )
+    assert response.status_code < 300, \
+        f'Removing release role from user API request failed with {response.status_code} and {response.text}'
+
+
+def user_removes_prerelease_user_roles_if_needed_via_api(user_emails: list) -> None:
+    allowed_users = [
+        'ees-prerelease1@education.gov.uk',
+        'ees-prerelease2@education.gov.uk',
+        'ees-prerelease3@education.gov.uk']
+
+    if len(set(user_emails).intersection(allowed_users)) < 2:
+        raise AssertionError('`User emails` must contain at least two allowed prerelease users')
```

Comment:
Why at least 2?  Could we not call this with just a singleton list?

Shows you've been working around code too long when you call a list with one item in it "singleton" :)

Reply:
No reason really, I've removed this restriction now